ast/term.go

```diff
@@ -919,9 +916,8 @@ func (ref Ref) Extend(other Ref) Ref {
 	head.Value = String(head.Value.(Var))
 	offset := len(ref)
 	dst[offset] = head
-	for i := range other[1:] {
-		dst[offset+i+1] = other[i+1]
-	}
+
+	copy(dst[offset+1:], other[1:])
```

Comment:
Here's my playground for fiddling with this... https://go.dev/play/p/oIwC0hpfau1 😄

Reply:
The switch to [`copy`](https://pkg.go.dev/builtin#copy) makes the intent a bit more explicit and concise, which I like. :smile: